/**:
  ros__parameters:
    backward_path_length: 5.0
    forward_path_length: 200.0
    backward_length_buffer_for_end_of_pull_over: 5.0
    backward_length_buffer_for_end_of_pull_out: 5.0
    minimum_pull_over_length: 16.0

    refine_goal_search_radius_range: 7.5

    # parameters for turn signal decider
    turn_signal_intersection_search_distance: 30.0
    turn_signal_intersection_angle_threshold_deg: 15.0
    turn_signal_minimum_search_distance: 10.0
    turn_signal_search_time: 3.0
    turn_signal_shift_length_threshold: 0.3
    turn_signal_on_swerving: true

    path_interval: 2.0

    visualize_maximum_drivable_area: true

    lateral_distance_max_threshold: 2.0
    longitudinal_distance_min_threshold: 3.0

    expected_front_deceleration: -1.0
    expected_rear_deceleration: -1.0

    expected_front_deceleration_for_abort: -2.0
    expected_rear_deceleration_for_abort: -2.5

    rear_vehicle_reaction_time: 2.0
    rear_vehicle_safety_time_margin: 2.0
